README.md

```diff
@@ -131,87 +136,112 @@ Look at the [Raiden Light Client SDK folder of this repository](./raiden-ts/READ
-- Look at the [documentation](https://docs.raiden.network/)
-- Learn more by watching explanatory [videos](https://www.youtube.com/channel/UCoUP_hnjUddEvbxmtNCcApg)
-- Read our blog posts on [Medium](https://medium.com/@raiden_network)
+- Browsing the [developer portal](http://developer.raiden.network).
+- Reading the [Raiden documentation](https://docs.raiden.network/).
+- Watching the explanatory [videos](https://www.youtube.com/channel/UCoUP_hnjUddEvbxmtNCcApg).
+- Reading our blog posts on [Medium](https://medium.com/@raiden_network).
 
 ## Try Out the Raiden Demo dApp
 
-The Raiden dApp is a demo and the first dApp to use SDK. It's a single page application (SPA) built on top of [Vue.js](https://vuejs.org/), [vuex](https://vuex.vuejs.org) and uses [vuetify](https://vuetifyjs.com) Material Design as UI framework.
+The Raiden dApp is a demo and the first dApp to be built on the SDK. It's a single page application (SPA) built on top of [Vue.js](https://vuejs.org/), [vuex](https://vuex.vuejs.org) and uses [vuetify](https://vuetifyjs.com) Material Design as UI framework.
 
-These step-by-step instructions will guide you through the process for trying out the Raiden demo dApp. The dApp is hosted at [https://lightclient.raiden.network/](https://lightclient.raiden.network/) and we will be using the Goerli testnet and MetaMask wallet in this example.
+These step-by-step instructions will guide you through the process for trying out the Raiden demo dApp.
+
+The dApp is hosted on [https://lightclient.raiden.network](https://lightclient.raiden.network/) and in this quick start guide we will be making a transfer using the Light Client on both mainnet and the Goerli testnet with the [MetaMask](https://metamask.io) wallet.
+
+Fot a more in depth user guide, see the [Light Client documentation](https://lightclient.raiden.network/docs/).
 
 ### Prerequisites
 
-You need to have MetaMask installed for your browser. If you don't have MetaMask, [visit their website](https://metamask.io/) to download and install it.
+- MetaMask needs to be installed in your browser and can be [downloaded from their website](https://metamask.io/).
+- An ETH balance. for mainnet you'll have to purchase ETH, for the Goerli testnet you can acquire some test ETH by:  
 
-### Steps to Make Your First Transfer
+  1. By visiting a Goerli faucet either at [https://faucet.goerli.mudit.blog](https://faucet.goerli.mudit.blog) or [https://goerli-faucet.slock.it/](https://goerli-faucet.slock.it/).
+  2. Follow the instructions on respective faucet on how to acquire the ETH.
 
-#### 1. Acquire ETH
+### Making a First Transfer
 
-1. Go to the Goerli faucet at [htts://faucet.goerli.mudit.blog](https://faucet.goerli.mudit.blog) or [https://goerli-faucet.slock.it/](https://goerli-faucet.slock.it/)
-2. Follow the instructions on how to acquire Goerli ETH on respective website
+1. **Connect to the dApp**  
 
-#### 2. Navigate to the Raiden Hub
+   - Mainnet  
```

Comment:
A good approach to have the two guides here :) I would make two separate sections as an outer braket. First for mainnet and second for testnet.

Reply:
Ok, I'll split it into two guides.